### PR TITLE
Doc 27 add security faq

### PR DESCRIPTION
**Pull request by jimlima**
Created security faq from text in word doc attached to ticket DOC-27 and added it to the documentation sidebar menu.

### CODE REVIEW

File: _data/docs.yml [hunk 1]
```diff
@@ -31,6 +31,8 @@
     - title: General
     - title: Universal Contact Tracing - Employee FAQ's
       keywords : ["tracking","wearable","wristband"]
+    - title: Microshar Data Security and Privacy FAQs
```
Comment (JulienBardin) [inline _data/docs.yml:34]: typo

File: docs/2/general/faq/security-faq.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+---
+layout: docs
+title: {Microshar Data Security and Privacy FAQs}
+description: {Frequently asked questions regarding Microshare data security and privacy}
```
Comment (JulienBardin) [inline docs/2/general/faq/security-faq.md:4]: typos and also shouldn't contain: '{}'

File: docs/2/general/faq/security-faq.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+{% capture title %}
+Is the Microshare application compliant with relevant data protection and privacy regulations, such as GDPR or SOC 2?
+{% endcapture %}
+
+{% capture content %}
+Microshare is SOC 2, IASME Cyber Essentials, and IASME Cyber Assurance (GDPR) complaint. We undergo annual third-party assessments as part of the recertification process to ensure we meet all data protection and privacy requirements. For more information, please visit the Microshare Data Security and Privacy Trust Center at https://www.microshare.io/microshares-trust-center-data-security/
+{% endcapture %}
+
+{% include accordion.html title=title content=content key="security" %}
```
Comment (JulienBardin) [inline docs/2/general/faq/security-faq.md:42]: the "key" elements should all be different.

File: docs/2/general/faq/security-faq.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+<!-- ********************
+**    New Question     **
+********************* -->
+
+{% capture title %}
+What security measures do you implement during the software development process?
+{% endcapture %}
+
+{% capture content %}
+Microshare follows an iterative SDLC and is implementing the MS Security Development Lifecycle (MS SDL) as a security-specific overlay to our processes. OWASP standards are followed for the development and configuration for web accessible applications in APIs. All code and infrastructure changes are tracked in a source control repository. Any changes must be reviewed/approved before they are allowed to be submitted to the main branch.  For more information on our secure development process please see the Microshare Security Letter at  https://www.microshare.io/wp-content/uploads/2022/08/Microshare-SOC-2-Type-2-Security-Letter.pdf
```
Comment (JulienBardin) [inline docs/2/general/faq/security-faq.md:84]: to put a link use "[]()" : [text appearing, could be the link](link)

**Review by JulienBardin — CHANGES_REQUESTED** [review]
A few changes necessary, @jtantum can you also review the content ?

File: _data/docs.yml (hunk 1 above)
Comment (jtantum) [reply _data/docs.yml]: Addressed.

File: docs/2/general/faq/security-faq.md (hunk 2 above)
Comment (jtantum) [reply docs/2/general/faq/security-faq.md]: Addressed.

File: docs/2/general/faq/security-faq.md (hunk 3 above)
Comment (jtantum) [reply docs/2/general/faq/security-faq.md]: Addressed.

File: docs/2/general/faq/security-faq.md (hunk 4 above)
Comment (jtantum) [reply docs/2/general/faq/security-faq.md]: Addressed.

**Comment by jtantum** [comment]
Hi @JulienBardin, @jimlima is on vacation this week so I made the needed changes that you pointed out. All the hyperlinks for this doc should be all set, made a few additional spelling/gramar changes too. Have reviewed and it now looks good to me. After your final review this should be ready to merge.

**Comment by jtantum** [comment]
@JulienBardin have adjusted the title accordingly. Let me know if there are any other changes that need be made!

**Comment by JulienBardin** [comment]
@jtantum I did the changes as the name of the file was still wrong with lower dash instead of dash, uppercase while it can't and spaces that can't be there.

**Pull request merged**